eox_tenant/tenant_wise/proxies.py

```diff
@@ -126,25 +124,23 @@ def get_value_for_org(cls, org, val_name, default=None):
         if cached_value:
             return cached_value
 
-        tenant_config = TenantConfig.objects.values_list("lms_configs")
-        microsite_config = Microsite.objects.values_list("values")  # pylint: disable=no-member
+        user = get_current_request().user
 
-        for config in chain(tenant_config, microsite_config):
-            try:
-                current = json.loads(config[0])
-                org_filter = current.get("course_org_filter", {})
-            except IndexError:
-                continue
+        if user.is_superuser:
```

Comment:
We can move this to the platform, specifically the view where we render the index